packages/cli/lib/common.js

```diff
@@ -14,7 +14,7 @@ function defaultCallback (error) {
   } else {
     console.log('hops: Server listening at ' + url.format({
       protocol: hopsConfig.https ? 'https' : 'http',
-      hostname: hopsConfig.host,
+      hostname: hopsConfig.host === '0.0.0.0' ? 'localhost' : hopsConfig.host,
```

Comment:
I've been asked about this pseudo-IP a couple of times - while technically correct-ish, it does prevent users from copying/pasting the URL.

https://en.m.wikipedia.org/wiki/0.0.0.0